src/assets/javascript/app/init.js

```diff
@@ -14,18 +14,17 @@ window.DI = window.DI || {};
 // Globally disable second clicks on the continue button for radio button pages,
 // But allow onSubmit scripts to overwrite this global behaviour
 
-const checkForOnSubmit = document.querySelector("form[onsubmit]");
```

Comment:
Were there any forms that did this? I can't think why they would, but someone obviously thought we needed it originally!

Reply:
It looks like two forms use onSubmit, not sure why though, might be some old lingering code